setup/debian/install-podman-rootless.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/sh
+# https://github.com/containers/podman/blob/main/docs/tutorials/rootless_tutorial.md
+
+##Get root permission##
+
+## Uncomment line below to allow debugging:
+#set -x
+
+## General variables ##
+inet_dev='eth0'
+nftables_config='/etc/nftables.conf'
+# User that will run the pods/containers:
+users='podman'
+# Rootful (0) or rootless (1) install. Default (1) recommended.
+rootless='1'
+
+apt update && apt-get update && apt upgrade -y
+apt install -y nftables \
+               sudo \
+               fuse-overlayfs
+
+echo "## Only enable temporarily as these are testing repositories that could break the system!!
+deb http://deb.debian.org/debian testing main non-free contrib
+deb http://deb.debian.org/debian unstable main non-free contrib" >> /etc/apt/sources.list
+
+apt update && apt-get update
+apt-get -y -f install libsemanage-common
+apt install -y podman/testing \
+               slirp4netns/testing \
+               aardvark-dns
+
+sed -i '/testing/d' /etc/apt/sources.list
+sed -i '/unstable/d' /etc/apt/sources.list
+
+apt update && apt-get update
+apt -y autoremove && apt -y autoclean && apt -y clean
+
+systemctl --now enable systemd-networkd
+systemctl --now start systemd-networkd
+
+# Remove existing .bashrc and add variables to get podman working.
+rm -rf /home/$users/.bashrc
+touch /home/$users/.bashrc
+echo "export XDG_RUNTIME_DIR=/run/user/$(id -u $users)
+export DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/$(id -u $users)/bus
+export DOCKER_HOST=unix:///run/user/$(id -u $users)/podman/podman.sock" >> /home/$users/.bashrc
+
+
+# enable user namespaces (cgroup)
+if [ "$rootless -eq 1" ]; then
+  # enable user namespaces (cgroup)
+  if grep -q user_namespace.enable=1 /etc/default/grub; then
+    echo "user_namespace already enabled in grub2"
+  else
+    if grep -q user_namespace.enable=0 /etc/default/grub; then
+      sed -i 's/user_namespace.enable=0/user_namespace.enable=1/g' /etc/default/grub
+      grub-mkconfig -o /boot/grub/grub.cfg
+    else
+      sed -i 's/GRUB_CMDLINE_LINUX="/GRUB_CMDLINE_LINUX="user_namespace.enable=1 /g' /etc/default/grub
+      grub-mkconfig -o /boot/grub/grub.cfg
+    fi
+  fi
+fi
+
+# enable binfmt to emulated foreign architectures
+systemctl start systemd-binfmt.service
```

Comment:
I think this isn't necessary because installing 'binfmt-support qemu-user-static' on Debian 10 will already enable/ start that service.. at least from my knowing

BTW. the two mentioned packages are part of the base package? I do not see that you have installed them.